lib/errors.js

```diff
@@ -9,6 +9,7 @@ function ValidationError(rule, params, info) {
     const valuePath = ['value'].concat(opts.path);
     const key = valuePath[valuePath.length - 1];
 
+    this.stack = (new Error()).stack;
```

Comment:
Oh yes, I meant to wrap this in a dev-only block, but I guess I'll just remove it and only add it for debug when needed.